tests/test_cmd_lossless.sh

```diff
@@ -68,6 +72,17 @@ pushd ${TMP_DIR}
   "${AVIFENC}" -y 400 -s 10 -l "${INPUT_GRAY_PNG}" -o "${ENCODED_FILE}"
   "${AVIFDEC}" "${ENCODED_FILE}" "${DECODED_FILE_LOSSLESS}"
   "${ARE_IMAGES_EQUAL}" "${INPUT_GRAY_PNG}" "${DECODED_FILE_LOSSLESS}" 0
+
+  # SVT-AV1 supports 4:2:0 lossless starting with version 3.0.0.
```

Comment:
Nit: This comment can be misinterpreted to mean that SVT-AV1 supported 4:4:4 lossless before version 3.0.0. It would be good to avoid this misinterpretation. Should we remove "4:2:0"?